internal/balancer/balancer.go

```diff
@@ -6,18 +6,17 @@ import (
 	"github.com/ydb-platform/ydb-go-sdk/v3/internal/conn"
 )
 
-// Balancer is an interface that implements particular load-balancing
-// algorithm.
-//
-// Balancer methods called synchronized. That is, implementations must not
-// provide additional goroutine safety.
-type Balancer interface {
-	// Next returns next connection for request.
-	Next(ctx context.Context, opts ...NextOption) conn.Conn
+type (
+	Balancer = *Config
```

Comment:
1. It is internal type - nobody outside backage can declare var with the type - then it is safe to change interface to struct
2. Interface can be nil, some code use nil balancer - it need pointer to struct
3. No rename Balancer in the commit - for smaller code changes. It will rename in next commit (with rename/move type changes only).